openslides/motions/signals.py

```diff
@@ -21,146 +21,132 @@ def setup_motion_config(sender, **kwargs):
     motions_workflow = ConfigVariable(
         name='motions_workflow',
         default_value='1',
-        form_field=forms.ChoiceField(
-            widget=forms.Select(),
-            label=ugettext_lazy('Workflow of new motions'),
-            required=True,
-            choices=[(str(workflow.pk), ugettext_lazy(workflow.name)) for workflow in Workflow.objects.all()]))
+        html_input_type='radio',
+        label=ugettext_lazy('Workflow of new motions'),
+        choices=[(str(workflow.pk), ugettext_lazy(workflow.name)) for workflow in Workflow.objects.all()])
```

Comment:
Then the autoupdate should send that the config has to be updated. This code is called every time you access the config api (except simple retrieving values from the cache).